fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -1819,6 +1823,63 @@ public ParseNode visitScalarSubquery(StarRocksParser.ScalarSubqueryContext conte
         return new BinaryPredicate(op, (Expr) visit(context.booleanExpression()), subquery);
     }
 
+    @Override
+    public ParseNode visitShowFunctionsStatement(StarRocksParser.ShowFunctionsStatementContext context) {
+        boolean isBuiltIn = context.BUILTIN() != null;
+        boolean isVerbose = context.FULL() != null;
+
+        String dbName = null;
+        if (context.db != null) {
+            dbName = getQualifiedName(context.db).toString();
+        }
+
+        String pattern = null;
+        if (context.pattern != null) {
+            pattern = ((StringLiteral) visit(context.pattern)).getValue();
+        }
+
+        Expr where = null;
+        if (context.expression() != null) {
+            where = (Expr) visit(context.expression());
```

Comment:
```suggestion
            where = visit(context.expression(), Expr.class);
```

Reply:
do you suggest use com.starrocks.sql.parser.AstBuilder#visit(List<? extends ParserRuleContext>, Class) ? it doen't not match args.